src/SFA.DAS.ProviderApprenticeshipsService.Application/Commands/SubmitCommitment/SubmitCommitmentCommand.cs

```diff
@@ -10,6 +12,8 @@ public class SubmitCommitmentCommand : IAsyncRequest
 
         public string Message { get; set; }
 
-        public string SaveOrSend { get; set; }
+        public AgreementStatus AgreementStatus { get; set; }
```

Comment:
A couple of questions here. Should we be passing ApiTypes to the Command Handler + Should the Command handler decide whether or not to create a task? Thoughts @mgwilliam ?